src/web/routes/application/flow-control/state-machine/selectors/selectors.js

```diff
@@ -1,10 +1,10 @@
-const { CHECK_ANSWERS_URL, TERMS_AND_CONDITIONS_URL } = require('../../../paths')
+const { CHECK_ANSWERS_URL, TERMS_AND_CONDITIONS_URL, prefixPath } = require('../../../paths')
```

Comment:
maybe rename to something like `addPrefixToPath`? I wasn't immediately sure what `prefixPath` wass.